application/objs/paginationtoken.go

```diff
@@ -0,0 +1,68 @@
+package objs
+
+import (
+	"fmt"
+
+	"github.com/MadBase/MadNet/application/objs/uint256"
+	"github.com/MadBase/MadNet/errorz"
+)
+
+type PaginationToken struct {
```

Comment:
Can nil value be made useful to help with preservation of for loop semantics above?

Reply:
Not entirely sure what you mean. Are you saying that instead of a nil paginationtoken we set it to just the prefix if it's not passed as to get rid of the edge case?